Client/Frontend/Home/HistoryHighlights/HistoryHighlightsViewModel.swift

```diff
@@ -141,8 +137,6 @@ class HistoryHighlightsViewModel {
     }
 
     func switchTo(_ highlight: HighlightItem) {
-        if urlBar.inOverlayMode { urlBar.leaveOverlayMode() }
```

Comment:
Just to make sure I understand, so now when will that happen? I think my question applies to changes in jump back in view model as well, probably the same answer to both.

Reply:
We don't need to dismiss the keyboard for this sections because this is considered a tab change so it will call `overlayManager.switchTab()`